opencodelists/settings.py

```diff
@@ -81,6 +81,8 @@ def get_env_var(name):
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
+TAKE_SCREENSHOTS = os.environ.get("TAKE_SCREENSHOTS", default=False) == "True"
```

Comment:
I like the use of a specific env var for taking screenshots rather than trying to overload IN_PRODUCTION. I also appreciate that you added your reasoning in the commit message.

Reply:
Thanks. I thought it was quite reasonable that you may want to run locally, and still have the feedback button operational.

Sticking everything into `IN_PRODUCTION` wouldn't allow that.